variables.tf

```
@@ -36,6 +36,15 @@ variable "assume_role_conditions" {
   default     = []
 }
 
+variable "assume_role_actions" {
+  type = set(object({
+    type        = string
+    identifiers = list(string)
+  }))
```

This should be set/list of string as we don't use complex objects here, like we did in conditions or principals:

```suggestion
  type = set(string)
```